### DISCUSSION TITLE
how to input C-c C-space in the keypad state

### DISCUSSION THREAD

**Discussion by capfredf** [meow-edit/meow#636] [Q&A] [answered]
The agda2 mode binds its most frequent command to `C-c C-SPC` in the vanilla Emacs. How do I invoke this command in the keypad state? I have tried a bunch of key sequences such as `SPC c SPC SPC`, `SPC c C-SPC`,  but none of them work.

**Comment by Esnos33** [comment]
I also didn't how to to do it, but I just use `meow-block`, `meow-change`, insert what I want, `meow-cancel-selection`, `agda2-load` with `SPC c SPC l`. You can also add to your init file
```
  (meow-leader-define-key 
   '("i" . "C-c C-SPC")
...
```
and with than you can run agda2-give with `SPC i`

**Comment by DogLooksGood** [comment, marked as answer]
You can't do it with keypad, because SPC is used as `Literal Prefix`. There isn't much key bindings you can't do, and `C-c C-SPC` is one of them.
